Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java

```diff
@@ -161,7 +153,8 @@ public String addConcept(HttpServletRequest req, Principal principal, Model mode
      * @throws IllegalAccessException
      */
     @RequestMapping(value = "/auth/conceptlist/addconceptwrapper/conceptsearch", method = RequestMethod.POST)
-    public String search(HttpServletRequest req, ModelMap model) throws LuceneException, IllegalAccessException {
+    public String search(@ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean, HttpServletRequest req,
```

Comment:
Added conceptWrapperAddBean to model object to be used as a commandObject in the jsp form.